src/nvim/screen.c

```diff
@@ -764,9 +764,7 @@ static void win_update(win_T *wp, Providers *providers)
   int i;
   long j;
   static bool recursive = false;  // being called recursively
-  const linenr_T old_botline = wp->w_botline;
-  const int old_wrow = wp->w_wrow;
-  const int old_wcol = wp->w_wcol;
+  int old_botline = wp->w_botline;
```

Comment:
don't see why the `const` or `linenr_T` declarations would be reverted. Basically, any non-functional style/conventional improvements should not be reverted.